### DISCUSSION TITLE
Previously Recognized Plots Missing from Farming

### DISCUSSION THREAD

**Discussion by DendyDarzartar** [Chia-Network/chia-blockchain#6201] [Support]
Hello, 

I am hoping someone can assist me with figuring out how to get my some of my plots recognized. Roughly half of them are not showing now after having been previously farming successfully and recognized by Chia.

The only recent change I can recall is being prompted to fix some vulnerabilities via a terminal prompt when I first started up Chia the night before last, which I had agreed to do. 

I have verified the directories that the plots are stored on were added to Chia, I have confirmed that the ubuntu OS recognizes the external hdd drives they are stored on and that it allows me to read and rename their contents, and I have tried renaming one of the non-recognized plots from .plot to .alot, moving it to the desktop, moving it back to the drive, and then renaming it back to .plot but that didnt allow it to be detected.

Just to give a bit more detail, I have 6 total hdd. Of the 6 only 3 are having their plots read despite all of them being properly listed in the display when I attempt to add their plot directories via the Chia gui.
 
Again they all show in the filesystem of the OS and are accessible from there as well, and of the 3 hdd that have plots that are being successfully read a few from each drive are not being read. 

All of the plots I have checked so far end in .plot and I would like to reiterate they were all farming without apparent issue and were showing in the GUI a few days ago.

I have no other plotting or farming machines and I, to my knowledge, have not changed any sort of IDs associated with the farmed plots or with my Chia wallet nor setup. (Aside from the aforementioned potential mistake of updating vulnerabilities I have made a point not to click anywhere or run any commands when I have not been 100% sure of what they do.)

I am a bit new to the ubuntu OS as well as to Chia so if you need a specific log file, please let me know where I might be able to find it and I will produce it for you. 

Thank you for taking the time to read this and I look forward to hopefully solving this mystery and getting these lost little plots back on their farm lol

**Comment by DendyDarzartar** [comment]
I was able to figure this out. 

I had to edit the config.yaml file for chia to add in some directories. 

It seems that the program i was using to automate plotting had a few directories in its own config.yaml file which caused it to create a new subfolder on the various external hdds. Those new folders were not in chia's config.yaml but the parent folders were hence chia missing some plots but seeing others off of the same drive.

Maybe this can help someone else.

**Comment by BrandtH22** [comment]
@DendyDarzartar , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
